app/ui/legacy/src/main/res/values/dimensions.xml

```diff
@@ -32,8 +32,4 @@
     <dimen name="floatingActionButtonMargin">16dp</dimen>
     <!-- Height of FloatingActionButton (56dp) plus two times floatingActionButtonMargin -->
     <dimen name="floatingActionButtonSpacing">88dp</dimen>
-
-    <dimen name="appTextSizeSmaller" format="dimension">14sp</dimen>
-    <dimen name="appTextSizeSmall" format="dimension">16sp</dimen>
-    <dimen name="appTextSizeMedium" format="dimension">18sp</dimen>
```

Comment:
There are a couple of places where we allow the user to configure the text size via *Settings → General settings → Display → Font size*. When using the value "Default" the views are not touched and whatever is set in the layout file will be used. This could lead to the default value not matching any of the other values the user can select (tiniest, tiny, smaller, small, medium, large, larger). To avoid this, we explicitly set the text size of those views in the layout file to one of the values the user can also manually select. That's what the `appTextSize*` values were used for.

Reply:
Right, that might need to be updated to match the Material text sizes then.

Reply:
#7920